### DISCUSSION TITLE
【活动报名中】新能源汽车数据基础设施专场 Meetup 来上海啦！！

### DISCUSSION THREAD

**Discussion by Wenjie0329** [GreptimeTeam/greptimedb#4298] [General]
在大数据时代，新能源汽车行业蓬勃发展。基于这一趋势，数据基础设施软件也在不断演变。企业数字化转型的深入推动了多云架构的繁荣，而汽车业务在多云多活应用中的表现如何？此外，汽车数据在边缘端和云端的低成本应用、Kafka 在多用户数据场景下的技术演进，以及车端嵌入式时序数据库面临的技术挑战也备受关注。这些话题将在新能源汽车专场 Meetup 中一一探讨，期待你的参与，共同见证技术的进步与创新。

GreptimeDB & AutoMQ Meetup 新能源汽车的数据基础设施

日期：2024年7月13日 14:30-17:30

地点：上海市徐汇区田林路192号1楼微软 Reactor 一楼

人数：50人

报名入口：https://1965290734055.huodongxing.com/event/4762909732100

![WechatIMG18431](https://github.com/GreptimeTeam/greptimedb/assets/117070618/e8a0d79c-5a69-4726-9677-23b6f5b7a194)

### 14:40-15:00

议题名称：AutoMQ 多云原生再升级：多模态共享存储架构

演讲人：周新宇，AutoMQ 联合创始人 & CTO

议题简介：

介绍 AutoMQ 的云原生架构及技术优势

探讨多云原生架构的挑战及应对方案

详解多云多模态共享存储架构及其业务价值

展望 AutoMQ 的未来发展和产品更新

### 15:00-15:30

议题名称：长城汽车多云多活架构的实践与探索

演讲人：陈天予，长城汽车技术总监

议题简介：

长城汽车多云多活架构的目标和挑战

多云多活架构的成本与技术权衡

为什么选择 AutoMQ 代替 Kafka

### 15:30-16:00

议题名称：One database, two sides: GreptimeDB 边云一体化数据库介绍

演讲人：罗傅聪，GreptimeDB 研发工程师

议题简介：

介绍 GreptimeDB 边云一体化数据库的架构和实现

探讨边缘端和云端的数据处理和整合

### 16:30-17:00

议题名称：小红书深度解读 AutoMQ 云原生及多云容灾架构

演讲人：黄章衡，小红书消息引擎研发专家

议题简介：

分享小红书 Kafka 应用的场景挑战和技术演进

深入解读 AutoMQ 云原生架构和多云容灾方案

### 17:00-17:30

议题名称：车载嵌入式时序数据库的技术挑战和方案

演讲人：黄磊，GreptimeDB 研发工程师

议题简介：

介绍嵌入式平台时序数据的采集、存储和分析需求

探讨 GreptimeDB 在嵌入式平台上的技术挑战和解决方案

_到场还有精美礼品哦~ 欢迎线下交流！_

**Comment by CodePrometheus** [comment]
@Wenjie0329 hello, are there any recorded videos about this meetup?

**Reply by Wenjie0329** [reply]
Yes, we have posted the recording to our wechat channel. You can search Greptime in WeChat Channel and view the livestream record. :)
